src/main.rs

```diff
@@ -83,9 +83,15 @@ fn main() {
     let options = {
         let mut options: Options = Docopt::new(USAGE)
             .and_then(|d| {
+                let mut args: Vec<_> = std::env::args().collect();
+                if args[1] != "outdated" {
```

Comment:
If this block is indeed added should check the length of `args` before trying to access the second item or else we'll panic if `args` length is 1. 

e.g.:

```rust
if args.len() > 1 && args[1] != "outdated" //snip
```

Reply:
Yes, very true!